lib/bemxjst/utils.js

```diff
@@ -7,6 +7,8 @@ var singleQuot = '&#39;';
 var matchXmlRegExp = /[&<>]/;
 
 exports.xmlEscape = function(string) {
+  if (!string)
```

Comment:
@miripiruni I will right test cases for all primitive values, so we can discuss them.

These are suitable places? :
```
test/bemcontext-attrescape-test.js
test/bemcontext-xmlescape-test.js
```